plugins/pagerduty/plugin.spec.yaml

```diff
@@ -23,7 +43,7 @@ hub_tags:
   features: []
 sdk: 
   type: full 
-  version: 5 
+  version: 6.1.0
```

Comment:
I think we could flip this from full -> slim? Moving from the old SDK (3.38 image full) to the new slim one will keep the OS the same. I swapped them around when I bumped the SDK at one point so that the slim named image was actually smaller.

[PR](https://github.com/rapid7/komand-plugin-sdk-python/pull/138/files#diff-a38774d770a0de9d4180e46a56396ecf9c54a90a4e4f10fb89bd4b90213e7177R1) this change that full 3-38 becomes slim-3

Reply:
Interesting! TY I wasn't aware of this